packages/blade/src/components/Input/BaseInput/getBaseInputStyles.ts

```diff
@@ -0,0 +1,27 @@
+import type { CSSObject } from 'styled-components';
+import type { Theme } from '~components/BladeProvider';
+import getTextStyles from '~components/Typography/Text/getTextStyles';
+import { makeBorderSize, makeSpace } from '~utils';
+
+const getBaseInputStyles = ({ theme }: { theme: Theme }): CSSObject => ({
+  ...getTextStyles({
```

Comment:
This is causing some visual issues on native. As per the current implementation the `lineHeight` on native is `24px` which looks visually weird. I tried making it `20px` and it seems much better. Attaching SS with both lineHeights. Can you confirm with design once and get the right lineHeight?

`24px`:
<img width="366" alt="Screenshot 2022-08-05 at 4 29 35 PM" src="https://user-images.githubusercontent.com/24487274/183069920-1158b404-f6b0-4652-9f75-8567fc2039cb.png">

`20px`:
<img width="369" alt="Screenshot 2022-08-05 at 4 29 28 PM" src="https://user-images.githubusercontent.com/24487274/183069964-725ecc9c-ca1d-498f-b3e5-320d807e24eb.png">

Reply:
I'm actually just rendering the body medium text and not something which is custom. let me double check it.

Reply:
@chaitanyadeorukhkar can you update here?

Reply:
Running on iOS. Sorry, didn't get any notifications for your comments until you tagged me 🤦

Reply:
Fixed in #632